src/frontend/js/utils/isSafeURI.ts

```diff
@@ -0,0 +1,66 @@
+import { location } from 'utils/indirection/window';
+
+const REDIRECT_WHITELIST =
+  (window as any).__richie_frontend_context__?.context?.redirect_whitelist || [];
```

Comment:
I don't find where you add this property in the context

Reply:
Related to the mistake in `src/richie/apps/core/context_processors.py`